src/views/app/CommunityLanding/index.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import HomeLayout from "../../../layout/HomeLayout.jsx";
 import CommunityLanding from "../../../pages/CommunityLanding/index.jsx";
+import Ambassador from "../../../components/ambassador/index.jsx";
```

Comment:
@senthil-k8s change ` ../../../components/ambassador/index.jsx` to ` ../../../components/Ambassador/index.jsx`